godot/Cargo.toml

```diff
@@ -18,6 +18,10 @@ double-precision = ["godot-core/double-precision"]
 experimental-godot-api = ["godot-core/experimental-godot-api"]
 experimental-threads = ["godot-core/experimental-threads"]
 experimental-wasm = []
+experimental-wasm-nothreads = [
+    "experimental-wasm",
+    "godot-core/experimental-wasm-nothreads",
+]
```

Comment:
I was thinking about adding a `compile_error!` to detect when both `experimental-wasm` and `experimental-wasm-nothreads` are specified, but since this is always done here, I don't think it's possible?

We would need to treat the two features separately and complicate some existing code in `godot` to use `#[cfg(any(...))]` instead :thinking:

Reply:
I think we can remove the dependency on `"experimental-wasm"` and instead gate the changes behind `all(target_family = "wasm", feature = "experimental-wasm-nothreads")`, which should be enough as `target_family = "wasm"` implies `feature = "experimental-wasm"` (we have a compiler error set in place if you use wasm without that feature).